arangod/RestServer/QueryRegistryFeature.cpp

```diff
@@ -174,6 +174,12 @@ void QueryRegistryFeature::prepare() {
 
 void QueryRegistryFeature::start() {}
 
+void QueryRegistryFeature::stop() {
+  TRI_ASSERT(_queryRegistry != nullptr);
+  _queryRegistry->disallowInserts();
```

Comment:
whats the differrence between calling `disallowInserts` on shutdown and checking `application_features::ApplicationServer::isStopping()` ? 
Seems to me like one or the other would be sufficient

Reply:
ok, I will unify it.